modules/apps/feature-flag/feature-flag-web/src/main/resources/META-INF/resources/js/FeatureFlagToggle.tsx

```diff
@@ -61,9 +65,10 @@ const FeatureFlagToggle = ({
 	return (
 		<>
 			<ClayToggle
+				aria-label={featureFlagTitle + featureFlagLabel}
```

Comment:
We can't join two language keys like this because it will cause them to be read differently for LTR and RTL languages. Basically for LTR it would read `LPS-12345 Disabled` and for RTL it would read `Disabled LPS-12345`.

There are two options to handle this... one is to join them using a language key like `x-x`, then the string can be properly inverted for RTL. The other method is to use another way of adding the extra information, like using `aria-describedby`. Something like [this example](https://github.com/liferay/liferay-portal/commit/3016c187ffcf9365ace51f6651792e60402b5689). I would recommend going for aria-describedby since I think it will be much simpler in this situation.